### PR TITLE
Acrolinx 9/1

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: docs/architecture/modern-web-apps-azure/test-asp-net-core-mvc-apps.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ Most of the integration tests in your ASP.NET Core apps should be testing servic
 
 For ASP.NET Core applications, the `TestServer` class makes functional tests fairly easy to write. You configure a `TestServer` using a `WebHostBuilder` (or `HostBuilder`) directly (as you normally do for your application), or with the `WebApplicationFactory` type (available since version 2.1). Try to match your test host to your production host as closely as possible, so your tests exercise behavior similar to what the app will do in production. The `WebApplicationFactory` class is helpful for configuring the TestServer's ContentRoot, which is used by ASP.NET Core to locate static resource like Views.
 
-You can create simple functional tests by creating a test class that implements IClassFixture\<WebApplicationFactory\<TEntry>> where TEntry is your web application's Startup class. With this in place, your test fixture can create a client using the factory's CreateClient method:
+You can create simple functional tests by creating a test class that implements `IClassFixture\<WebApplicationFactory\<TEntry>>`, where `TEntry` is your web application's `Startup` class. With this in place, your test fixture can create a client using the factory's `CreateClient` method:
 
 ```cs
```
Comment (Youssef1313) [inline docs/architecture/modern-web-apps-azure/test-asp-net-core-mvc-apps.md:155]: A bit unrelated to the PR:

~~~suggestion
```csharp
~~~

File: docs/architecture/modern-web-apps-azure/test-asp-net-core-mvc-apps.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ public class BasicWebTests : IClassFixture<WebApplicationFactory<Startup>>
 }
 ```
 
-Frequently, you'll want to perform some additional configuration of your site before each test runs, such as configuring the application to use an in memory data store and then seeding the application with test data. To do this, you should create your own subclass of WebApplicationFactory\<TEntry> and override its ConfigureWebHost method. The example below is from the eShopOnWeb FunctionalTests project and is used as part of the tests on the main web application.
+Frequently, you'll want to perform some additional configuration of your site before each test runs, such as configuring the application to use an in memory data store and then seeding the application with test data. To do this, create your own subclass of `WebApplicationFactory\<TEntry>` and override its `ConfigureWebHost` method. The example below is from the eShopOnWeb FunctionalTests project and is used as part of the tests on the main web application.
 
 ```cs
```
Comment (Youssef1313) [inline docs/architecture/modern-web-apps-azure/test-asp-net-core-mvc-apps.md:171]:
~~~suggestion
```csharp
~~~

**Review by BillWagner — APPROVED** [review]
These all look great @gewarren 
You can :shipit: when ready.

**Comment by gewarren** [comment]
Thanks for the reviews!

**Pull request merged**